api/src/main/java/org/apache/iceberg/catalog/Catalog.java

```diff
@@ -28,7 +29,7 @@
 /**
  * Top level Catalog APIs that supports table DDLs and namespace listing.
  */
-public interface Catalog {
+public interface Catalog extends Closeable {
   /**
    * creates the table or throws {@link AlreadyExistsException}.
```

Comment:
nit: what about having the same formatting for Javadoc in this file? I see that some comments start with a capital letter some not.